hedera-node/hedera-file-service-impl/src/test/java/com/hedera/node/app/service/file/impl/test/handlers/FileDeleteTest.java

```diff
@@ -98,26 +114,28 @@ void fileIdNotFound() throws PreCheckException {
         mockPayerLookup();
         given(mockStore.getFileMetadata(notNull())).willReturn(null);
         final var context = new FakePreHandleContext(accountStore, newDeleteTxn());
-        context.registerStore(ReadableFileStoreImpl.class, mockStore);
+        context.registerStore(ReadableFileStore.class, mockStore);
 
         // when:
         assertThrowsPreCheck(() -> subject.preHandle(context), INVALID_FILE_ID);
     }
 
     @Test
-    @DisplayName("File without keys returns error")
-    @Disabled("Revisit after fix for https://github.com/hashgraph/hedera-services/issues/7646")
-    void noFileKeys() throws PreCheckException {
```

Comment:
Shouldn't this still be present, to test the immutable case?

Reply:
on pre-handle it doesn't throw.
The one you are referring to that I deleted it is also for prehandle ``` void noFileKeys() throws PreCheckException```

Reply:
Ahh.  Should add the immutability check as a separate test, then.

Reply:
ok sure will be done. I will commit soon